requirements.txt

```diff
@@ -1,6 +1,6 @@
 ipython
 lockfile
-python-openflow
+python-openflow==2017.1b1.dev1
```

Comment:
Because it wil install 2017.1b1. This is because the version string is wrong. It should be either 2017.1b1.post1 or 2017.b2.dev1. @beraldoleal, do you want to correct this or we just pay more attention to next releases?